src/NzbDrone.Core.Test/ParserTests/QualityParserFixture.cs

```diff
@@ -119,6 +119,7 @@ public void should_parse_webdl480p_quality(string title, bool proper)
         [TestCase("Survivorman.The.Lost.Pilots.Summer.HR.WS.PDTV.x264-DHD", false)]
         [TestCase("Victoria S01E07 - Motor zmen (CZ)[TvRip][HEVC][720p]", false)]
         [TestCase("flashpoint.S05E06.720p.HDTV.x264-FHD", false)]
+        [TestCase("【极影字幕社】 ★1月新番【巴哈姆特之怒：马纳利亚朋友】【Shingeki no Bahamut Manaria Friends】【01】GB MP4_720", false)]
```

Comment:
Do you have other examples of this? Coz to me it just looks like a bad release name. We can't accept just '720' which is season 7 episode 20 in some cases, or 720 abs nr for certain korean shows.

Reply:
Looks like they are the only releasing group using this format.. I'll drop this commit